contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2345,13 +2345,8 @@ bbf_table_var_lookup(const char *relname, Oid relnamespace)
 			continue;
 
 		tbl = (PLtsql_tbl *) estate->datums[n];
-		if (strcmp(relname, tbl->refname) == 0)
+		if (strcmp(relname, tbl->refname) == 0 && tbl->tblname)
```

Comment:
is this necessary for the change?

Reply:
Yeah, without this change we get an error when trying to use TVFs that have table names which are too long. I think that the FIXME has already been fixed in the ANTLR parser - I didn't see any crashes or other unexpected regressions in the full JDBC test suite after making this change